docker/Dockerfile

```diff
@@ -14,10 +14,35 @@ RUN git config --global http.sslverify false
 RUN go build -o bin/go-dmux
 
 FROM golang:1.12-stretch as deploy
+
+RUN apt-get update && \
+    apt-get install -y procps && \
+    apt-get install -y ngrep && \
+    apt-get install netcat -y
+
+RUN apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
 WORKDIR /app
+
 COPY --from=build /app/bin/go-dmux .
-COPY --from=build /app/docker/config/conf.json ./config/conf.json
+COPY --from=build /app/docker/config/conf.json config/conf.json
```

Comment:
Let's remove this, we will mount necessary configs at runtime.

Reply:
ok